test/unit/models/future/policy_test.rb

```diff
@@ -22,5 +22,15 @@ class PolicyTest < ActiveSupport::TestCase
       assert_equal policy_1["content_id"], future_policy.content_id
       assert_equal policy_1["title"], future_policy.title
     end
+
+    test "#parent_policies" do
```

Comment:
Would be helpful for future-developer if this said what the method was expecting. e.g. `test "#parent_policies returns the policy areas associated withe the policy"`